.eslintignore

```diff
@@ -0,0 +1,2 @@
+coverage/
+testing-env/
```

Comment:
Instead of linting 2-3 specific directories, we'll lint EVERYTHING, unless it's in .eslintignore. Caught a few minor ESLint nits in unlinted files. :eyes: